examples/images_contours_and_fields/irregulardatagrid.py

```diff
@@ -0,0 +1,105 @@
+"""
+=======================================
+Contour plot of irregularly spaced data
+=======================================
+
+Comparison of a contour plot of irregularly spaced data interpolated
+on a regular grid versus a tricontour plot for an unstructured triangular grid.
+
+Since :meth:`~.axes.Axes.contour` and :meth:`~.axes.Axes.contourf` expect the
+data to live on a regular grid, plotting a contour plot of irregularly spaced
+data requires different methods. The two options are:
+
+* Interpolate the data to a regular grid first. This can be done with on-borad
+  means, e.g. via `~.tri.LinearTriInterpolator` or using external functionality
+  e.g. via `scipy.interpolate.griddata`. The latter is more versatile
+  as it allows for different kinds of interpolation (e.g. cubic). Then plot the
```

Comment:
Not true. The file `triinterpolator.py` that contains `LinearTriInterpolator` also contains `CubicInterpolator`, for example, a mere 56 lines later.